src/frontend/typer/provider.ts

```diff
@@ -0,0 +1,10 @@
+import { IFelt, ValiderIFelt, Valideringsstatus } from './felt';
+
+export const lagInitiellFelt = <T>(verdi: T, valideringsfunksjon: ValiderIFelt<T>): IFelt<T> => {
+    return {
+        feilmelding: 'Feltet er påkrevd, men mangler input',
```

Comment:
Maybe we run the valideringsfuksjon here to generate a sensible feilmelding, since for some fields empty input can still be valid.

Reply:
When we need the functionality for "not required" fields, we can adjust the function.